### ISSUE TITLE
How can I build this project without AWS CodeBuild?

### ISSUE THREAD

**Issue by jamiekang** [aws/sagemaker-mxnet-serving-container#101]
Hello,
I am trying to build this within linux (Amazon EC2) command line.
How can I run "buildspec.yml" in terminal? Anytool to convert buildspec.yml into sh script?

Thanks.

**Comment by nadiaya** [comment]
"buildspec.yml" is used for triggering and running tests on PRs and contains a list of shell commands to do so.

Could you give us more details about your use case and what are you trying to achieve?

**Comment by jamiekang** [comment]
**TL;DR:** the `Dockerfile.gpu` should be modified. (maybe the `Dockerfile.cpu` too)
It's trying to copy `sagemaker_mxnet_inference.tar.gz` but it should be `sagemaker_mxnet_serving_container.tar.gz`.

I was about to build my image as guided by the "Building Images" section of README.rst. (Env: python3, gpu(ec2 p3), mxnet 1.6.0, no EI) But I failed in running "docker build -t preprod-mxnet-serving:1.6.0-gpu-py3 -f Dockerfile.gpu ." command.

This is the error message I got:
```
Step 15/26 : WORKDIR /
 ---> Using cache
 ---> d504cb460349
Step 16/26 : COPY sagemaker_mxnet_inference.tar.gz /sagemaker_mxnet_inference.tar.gz
COPY failed: stat /var/lib/docker/tmp/docker-builder900894127/sagemaker_mxnet_inference.tar.gz: no such file or directory
```

I found the reason is because of some missing commands like these specified in buildspec.yml:
      - cp dist/sagemaker_mxnet_inference.tar.gz docker/$FRAMEWORK_FULL_VERSION/py3/sagemaker_mxnet_inference.tar.gz
      - cp -r docker/artifacts/* docker/$FRAMEWORK_FULL_VERSION/py3

But your answer is that the file is for test.
So I did more experiment and found the Dockerfile.gpu should be modfied as in TL;DR:

**Comment by nadiaya** [comment]
The latest dockerfiles are correct as the package name has been changed from  `sagemaker_mxnet_serving_container.tar.gz` to `sagemaker_mxnet_inference.tar.gz`.

I will update outdated documentation.

**Comment by jamiekang** [comment]
Thanks!
